src/Plots.jl

```diff
@@ -106,6 +106,15 @@ export
 
 # ---------------------------------------------------------
 
+import NaNMath
+# define functions (e.g. `_extrema`, that ignores NaNs, when the type is applicable. To overcome the destructive effects of https://github.com/JuliaLang/julia/pull/12563
+for fun in (:extrema, :minimum, :maximum)
```

Comment:
I know, right? It just feels like you should use the classic metaprogramming example when there is a chance :-) Oh well.